steampy/client.py

```diff
@@ -164,10 +164,10 @@ def get_trade_offer(self, trade_offer_id: str, merge: bool = True) -> dict:
 
     @login_required
     def accept_trade_offer(self, trade_offer_id: str) -> dict:
-        partner = self._fetch_trade_partner_id(trade_offer_id)
-        if partner == False:
-            print("Account on hold, cannot accept trades")
-            return None
+        try:
+            partner = self._fetch_trade_partner_id(trade_offer_id)
+        except SevenDaysHoldException:
+            return
```

Comment:
Just self._fetch_trade_partner_id(trade_offer_id)
Exception should be caught outside accept_trade_offer method